packages/data-point/lib/reducer-list/resolve.test.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-env jest */
```

Comment:
Most of this was existing code from the transform-expression directory. I used git mv to rename the transform-expression directory to reducer-list, but maybe I did something wrong because it's showing this as all new changes.

Reply:
thats odd and sad, looses history of other people's changes

Reply:
maybe its Github? @paulmolluzzo any experience with how a moved file would look in GitHub? @raingerber can you check on your local history see if its being interpreted as a move or a new file?

Reply:
No direct experience, sorry. The links below seem to provide some guidance though. We shouldn't lose the history, but we will have a "wall" in the blame where it'll look like this commit is responsible for all the code in the file. You will have to go to the parent commit (and that commits parent) to traverse history.

https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history

https://stackoverflow.com/questions/10828267/can-git-restructure-my-folders-without-losing-history

Sorry I can't be of more help. 😞